Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -2,26 +2,22 @@
 import io
 import json
 import re
-import ssl
 from datetime import datetime, timedelta
 
-import dateparser  # type: ignore
-import demistomock as demisto  # noqa: F401
-import pytz  # type: ignore[import]
+import dateparser
+import demistomock as demisto
+import pytz
 import requests
 import splunklib.client as client
 import splunklib.results as results
-import urllib2
+from splunklib.data import Record
 import urllib3
 from CommonServerPython import *  # noqa: F401
 from splunklib.binding import AuthenticationError, HTTPError, namespace
-from StringIO import StringIO
 
-urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+urllib3.disable_warnings()
 
 # Define utf8 as default encoding
-reload(sys)
-sys.setdefaultencoding('utf8')  # pylint: disable=maybe-no-member
 params = demisto.params()
 SPLUNK_TIME_FORMAT = "%Y-%m-%dT%H:%M:%S"
 VERIFY_CERTIFICATE = not bool(params.get('unsecure'))
```

Comment:
how us this being handled in the new version?

Reply:
fixed. we will use the requests handler if user does not want to verify certificate